core/includes/install.core.inc

```diff
@@ -1741,6 +1741,12 @@ function install_check_requirements($install_state) {
         $writable = backdrop_verify_install_file($settings_file, FILE_READABLE|FILE_WRITABLE);
         $exists = TRUE;
       }
+      if ($settings_file !== './settings.php') {
+        if (copy('./settings.php', $settings_file) === TRUE) {
+          $exists = TRUE;
+          $writable = backdrop_verify_install_file($settings_file, FILE_READABLE|FILE_WRITABLE);
+        }
+      }
```

Comment:
This check doesn't look right to me. It appears as though this would call `backdrop_verify_install_file()` twice on a non-root settings.php file. In addition it seems like if you already copied `settings.php` into a multisite directory, it would end up getting overwritten by the settings.php file from the root.